feed/templates/post_detail.html

```diff
@@ -6,8 +7,8 @@
     <div class="col-md-8 card mb-4  mt-3 left  top">
       <div class="card-body">
         <h1>{% block title %} {{ object.title }} {% endblock title %}</h1>
-        <p class=" text-muted">{{ post.author }} | {{ post.created_on }}</p>
-        <p class="card-text ">{{ object.content | safe }}</p>
+        <p class=" text-muted">{{ post.author }}  {{ post.created_on }}</p>
+        <p class="card-text ">{{ object.content | safe }} <img src="{{ object.image.url }}"></p>
```

Comment:
Is the object name `post` or `object` in the template? Both are being used and I havent tested this code out yet.

Reply:
i did that when I was adding the images. it works currently the images show up.

Reply:
So then `post.author` shouldn't work. You'll need to change the line above (I think) to `{{ object.author }}` and `{{ object.created_on }}`